application/templates/cms/search_data_source.html

```diff
@@ -0,0 +1,68 @@
+{% extends "base.html" %}
+{% from "_shared/_breadcrumb.html" import breadcrumb %}
+
+{% block pageTitle %}Search for an existing data source{% endblock %}
+
+{% set breadcrumbs =
+  [
+    {"url": url_for('static_site.topic', topic_slug=topic.slug), "text": topic.title}
```

Comment:
Feels like a breadcrumb back to the measure would be good as well?

Reply:
Opps, I'd meant to add a back link instead. Not sure we should add it to the breadcrumb as the measure title already appears as a caption before the title (so that it's super clear which measure you'd be adding a data source to).